src/features/MoviesBrowser/PersonDetailPage/PersonDetailsTile/index.js

```diff
@@ -1,22 +1,42 @@
 import { useSelector } from "react-redux";
 import { selectDetailItem } from "../../moviesBrowserSlice";
+import {
+    PersonDetailTile,
+    Content,
+    PersonName,
+    Subtitle,
+    StyledDetails,
+    Description,
+    StyledPoster,
+} from "./styled";
 
 const PersonDetailsTile = () => {
     const person = useSelector(selectDetailItem);
     return (
-        <div>
-            <h4>Person Details Tile</h4>
-            
-            <p>Name: {person.name}</p>
-            
-            <p> Birthday:  {person.birthday} </p>
-            
-            <p> Place of birth:  {person.place_of_birth} </p>
-            
-            <p> Image : <img src={`https://image.tmdb.org/t/p/w200${person.profile_path}`} alt={person.name} /></p>
-            
-            <p>Bio: {person.biography}</p>
-        </div>
+
+        <PersonDetailTile>
+            <StyledPoster
+                src={`https://image.tmdb.org/t/p/w400${person.profile_path}`}
```

Comment:
dał bym chyba jeszcze większy obrazek,  żeby nie było pixelozy

Reply:
Wydaje mi się, że jeszcze mógłby być lekko w lewo (wg projektu te obszary pomiędzy poster a tile są równe i mają , ale jak ustawiam margin w lewo, to mi się przesuwają również napisy i zlewają się ze zdjęciem. Zmienniłam margin w description na 40px; podtytułach 40px, ale znowu mi się zlewa PersonName. Jeszcze nie wiem jak to rozwiązać, bo już różne rozwiązania przetestowałam.